src/main/java/mate/academy/spring/controller/MovieController.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.dto.MovieRequestDto;
+import mate.academy.spring.dto.MovieResponseDto;
+import mate.academy.spring.mapper.MovieMapper;
+import mate.academy.spring.service.MovieService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/movies")
+public class MovieController {
+    private final MovieService movieService;
+    private final MovieMapper movieMapper;
+
+    public MovieController(MovieService movieService, MovieMapper movieMapper) {
+        this.movieService = movieService;
+        this.movieMapper = movieMapper;
+    }
+
+    @GetMapping
+    public List<MovieResponseDto> getAllMovies() {
+        return movieService.getAll().stream()
+                .map(movieMapper::parse)
+                .collect(Collectors.toList());
+    }
+
+    @PostMapping
+    public MovieResponseDto addMovie(@RequestBody MovieRequestDto requestDto) {
```

Comment:
```suggestion
    public MovieResponseDto add(@RequestBody MovieRequestDto requestDto) {
```
don't make sense to add entity name to the method names, from the context you understand to which entity these operations related